pkg/cli/helm/radiusclient.go

```diff
@@ -141,6 +141,13 @@ func AddRadiusValues(helmChart *chart.Chart, options *RadiusOptions) error {
 	}
 
 	for _, arg := range options.SetFileArgs {
+		// Continue to support: c:\\testdir\\testfile.crt
+		// and convert: c:\testdir\testfile.crt to c:/testdir/testfile.crt
+
+		if !strings.Contains(arg, "\\\\") && strings.Contains(arg, "\\") {
```

Comment:
Should we check for windows first? `\` has a different meaning in linux paths. It's a pretty rare case but `\` is legal as part of a filename in linux.

Reply:
Even better we could check [os.PathSeparator](https://cs.opensource.google/go/go/+/refs/tags/go1.22.2:src/os/path_unix.go;l=10)

Reply:
How about adding an additional check to ` runtime.GOOS`.
This along with the `os.PathSeparator` to stop using magic strings in the search.

There is precedence of this being used within the project

https://github.com/radius-project/radius/blob/74a3f25de96c0cee2b689fe33c6f3499e725c02b/pkg/cli/tools/util.go#L24

Reply:
@rynowak clean up the logic... tested on both Windows and Linux.

<img width="949" alt="image" src="https://github.com/radius-project/radius/assets/3308167/18e71da3-bae8-43b1-8dad-9464463a81c9">

Reply:
My first time seeing [filepath.ToSlash](https://pkg.go.dev/path/filepath#ToSlash).

Fix looks great!